### PR TITLE
Adding-Documentation

### PR DESCRIPTION
**Pull request by T19700**
Created a CONTRIBUTING.md file and revamped the README.md file to make it more user friendly and accessible for people.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Contributions
+
+### Community Guidelines
+
+Hi there! Welcome to the Data Structures and Algorithms Project created by shree1999! Here you will find the guidelines for how to contribute to this project.
+
+1. Please fork the project into your own repository.
+2. Make a clean branch to work off of
+     - Be specific when naming your branch
+3. Thoroughly terst and validate your additions before making a pull request
```
Comment (shree1999) [inline CONTRIBUTING.md]: Hi @T19700 a small spelling mistake `terst` should be `test`

**Review by shree1999 — COMMENTED** [review]
A very small spelling mistake but overall awesomely done...

File: CONTRIBUTING.md (hunk 1 above)
Comment (shree1999) [reply CONTRIBUTING.md]: On The contributors list I think we can do some automation to get the contributor's list and put it in README.md file 
Let me take this up.

Comment (T19700) [reply CONTRIBUTING.md]: I think I fixed the spelling error now! Good catch!

**Review by shree1999 — APPROVED** [review]

**Pull request merged**